### DISCUSSION TITLE
This list is very low quality

### DISCUSSION THREAD

**Discussion by thecatontheceiling** [awesome-windows11/windows11#16] [General]
Most of the stuff listed here simply does not work properly or breaks the operating system. I believe a thorough clean up needs to be done on the list as right now it is filled with garbage 

For example, Sycnex's debloater has been outdated for years and it causes major stability issues on latest versions of windows.

**Comment by jestxfot** [comment]
Check the README in the latest version of your browser, the file should work correctly
